### ISSUE TITLE
problem with read notifications

### ISSUE THREAD

**Issue by Gabriele69** [ubports/telegram-app#199]
This has happened on my device for some time: I compose a message. I send the message and the box appears blue (message sent and not yet read) but immediately the box changes color and turns green (message read). This happens even if the person who receives it has not read the message.
I'm sure of this also for having checked it on PC at the same time, and the message was still to be read. In addition, the time of the last access to the application of who should have read the message, is prior to the notification of reading
I have a BQ E4.5 15.04 RC and I use the Telegram App v2.5.4 (0)

**Comment by Flohack74** [comment]
Yes this is a confirmed bug. Unfortunately it looks like as if Telegram server sends the message flag set to read. I am not sure we can do much about it.

**Comment by Flohack74** [comment]
![image](https://user-images.githubusercontent.com/16186470/45488359-72c4a200-b761-11e8-8596-9bc1bfe8954e.png)
